internal/cmd/kafka/command_link_create.go

```diff
@@ -167,36 +190,129 @@ func (c *linkCommand) create(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+var printableLinkConfigs = map[string]bool{
+	"bootstrap.servers":                 true,
+	"connection.mode":                   true,
+	"link.mode":                         true,
+	"consumer.offset.sync.enable":       true,
+	"consumer.offset.group.filters":     true,
+	"consumer.offset.sync.ms":           true,
+	"consumer.group.prefix.enable":      true,
+	"acl.sync.enable":                   true,
+	"acl.filters":                       true,
+	"acl.sync.ms":                       true,
+	"auto.create.mirror.topics.enable":  true,
+	"auto.create.mirror.topics.filters": true,
+	"cluster.link.prefix":               true,
+	"topic.config.sync.ms":              true,
+	"topic.config.sync.include":         true,
+	"mirror.start.offset.spec":          true,
+	"security.protocol":                 true,
+	"sasl.mechanism":                    true,
+	"local.security.protocol":           true,
+	"local.sasl.mechanism":              true,
+}
+
+func linkConfigsCommandOutput(linkConfig map[string]string) string {
+	filtered := make(map[string]string)
+	for key, val := range linkConfig {
+		if _, ok := printableLinkConfigs[key]; ok {
+			filtered[key] = val
+		}
+	}
+	return properties.CreateKeyValuePairs(filtered)
+}
+
 func getJaasValue(apiKey, apiSecret string) string {
 	return fmt.Sprintf(`%s username="%s" password="%s";`, jaasConfigPrefix, apiKey, apiSecret)
 }
 
-func (c *linkCommand) getConfigMapAndLinkMode(configFile string) (map[string]string, linkMode, error) {
+func (c *linkCommand) getConfigMapAndLinkMode(configFile string) (map[string]string, *linkModeMetadata, error) {
 	if configFile != "" {
 		var linkMode linkMode
 		configMap, err := properties.FileToMap(configFile)
 		if err != nil {
-			return nil, linkMode, err
+			return nil, nil, err
 		}
 		linkModeStr, ok := configMap["link.mode"]
 		if !ok {
 			// Default is destination if no config value is provided.
 			linkMode = Destination
-		} else if strings.EqualFold(linkModeStr, "DESTINATION") {
-			linkMode = Destination
-		} else if strings.EqualFold(linkModeStr, "SOURCE") {
-			linkMode = Source
+			linkModeStr = DESTINATION
 		} else {
-			return nil, linkMode, errors.Errorf(`unrecognized link.mode "%s". Use DESTINATION or SOURCE.`, linkModeStr)
+			switch linkModeStr {
+			case DESTINATION:
+				linkMode = Destination
+				linkModeStr = DESTINATION
+			case SOURCE:
+				linkMode = Source
+				linkModeStr = SOURCE
+			case BIDIRECTIONAL:
+				linkMode = Bidirectional
+				linkModeStr = BIDIRECTIONAL
```

Comment:
Redundant, since it's already set.
```suggestion
			case DESTINATION:
				linkMode = Destination
			case SOURCE:
				linkMode = Source
			case BIDIRECTIONAL:
				linkMode = Bidirectional
```